test/jsonSchema.test.ts

```diff
@@ -152,6 +174,48 @@ describe('Json Schema Conversion Test File', function () {
         },
       ],
     },
+    time_field: {
+      value: 1726067527,
+      unit: 'MILLIS',
+      isAdjustedToUTC: true,
+    },
+  };
+
+  const row1FromParquetFile = {
+    string_field: 'string value',
+    int_field: 10n,
+    number_field: 2.5,
+    timestamp_array_field: { list: [{ element: new Date('2023-01-01 GMT') }] },
+
+    timestamp_field: new Date('2023-01-01 GMT'),
+
+    array_field: {
+      list: [{ element: 'array_field val1' }, { element: 'array_field val2' }],
+    },
+
+    obj_field: {
+      sub1: 'obj_field_sub1 val',
+      sub2: 'obj_field_sub2 val',
+    },
+
+    struct_field: {
+      list: [
+        {
+          element: {
+            sub8: {
+              list: [{ element: 'val1' }, { element: 'val2' }],
+            },
+            sub3: 'struct_field_string val',
+            sub4: 'struct_field_string val',
+            sub5: {
+              sub6: 'struct_field_struct_string1 val',
+              sub7: 'struct_field_struct_string2 val',
+            },
+          },
+        },
+      ],
+    },
+    time_field: 1726067527,
```

Comment:
Time field from parquet file will be either of two primitive types `INT32' for MILLIS and `INT64` for rest two

Reply:
That means you can write it using TIME_MILLIS but it'll never be read that way?

Reply:
I believe thats what spec is suggesting to map the logical type to a primitive type on write, however schema has the information of what type was set for this field.

I will double check to be sure